vc/authorization-credential-v1.jsonld

```diff
@@ -24,8 +24,8 @@
          "@id":"ex:issuerDIDDoc",
          "@type":"DIDDoc"
       },
-      "rpDIDDoc":{
-         "@id":"ex:rpDIDDoc",
+      "relyingPartyDIDDoc":{
```

Comment:
As this authorization credential is not coupled to OIDC/OAuth, we might want to use the [DID core spec](https://www.w3.org/TR/did-core/) language:
`requestingParty`

Reply:
I'd say we align ourselves with UMA (where it's also "requestingParty")

Reply:
Agree.

Reply:
@troyronda changed to `requestingPartyDID` but thinking about this has resurfaced old thoughts and ideas that you and I may have discussed - I opened several issues: #23, #24, #25